assets/i18n/id.json

```diff
@@ -25,245 +24,238 @@
     "dgc-exchanges-modal-anxpro-desc": " (USD, EUR, GBP, AUD, CAD, HKD, SGD, NZD)", 
     "dgc-exchanges-modal-localdogecoin": "LocalDogecoin",
-    "dgc-exchanges-modal-localdogecoin-desc": " (Sebuah website untuk membeli Dogecoin secara lokal.)",
-    "dgc-exchanges-modal-mining": "Kamu juga dapat mendapatkan Dogecoin dengan cara \"menambang\" Dogecoin. Menambang adalah ketika kamu menggunakan komputermu untuk memproses transaksi Dogecoin yang mana transaksi tersebut dilakukan oleh orang lain, dan kamu mendapatkan Dogecoin sebagai gantinya.", 
-    "dgc-exchanges-modal-mining-howto": "Mining ditujukan untuk pengguna yang sudah advance dikarenakan memerlukan banyak pengetahuan teknis untuk menambang. Jika kamu tertarik, <a class=\"btn btn-sm btn-outline-secondary\" href=\"https://dogecointutorial.com/mining.html\" target=\"_blank\">baca lebih lanjut disini</a>.", 
-    "dgc-exchanges-modal-disclaimer": "<strong>Catatan:</strong> TFaucet dan bursa online ini tidak dimiliki atau dikelola oleh Dogecoin. Meskipun kami yakin bahwa layanan tersebut dapat dipercaya dan dapat diandalkan - harap gunakan dengan risiko Anda sendiri.", 
+    "dgc-exchanges-modal-localdogecoin-desc": " (Sebuah website untuk membeli Dogecoin lokal.)",
+    "dgc-exchanges-modal-mining": "Kamu juga bisa mendapatkan Dogecoin dengan \"menambang\"nya. Penambangan ialah saat kamu menggunakan komputer untuk memproses transaksi Dogecoin orang-orang, dan sebagai imbalannya kamu bisa mendapatkan Dogecoin.", 
+    "dgc-exchanges-modal-mining-howto": "Penambangan hanya ditujukan untuk pengguna tingkat lanjut, karena untuk melakukannya membutuhkan banyak pengetahuan teknis. Jika kamu tertarik, <a class=\"btn btn-sm btn-outline-secondary\" href=\"https://dogecointutorial.com/mining.html\" target=\"_blank\">baca selengkapnya disini</a>.", 
+    "dgc-exchanges-modal-disclaimer": "<strong>Catatan:</ strong> Faucet dan penukaran daring tersebut tidak dimiliki atau dikelola oleh Dogecoin. Sementara kami puas dan mempercayainya, karena pelayanannya dapat diandalkan - mohon berhati-hati dan resiko ditanggung sendiri.", 
     "dgc-stores-modal-title": "TOKO:", 
-    "dgc-stores-modal-shop-online": "Kamu dapat membeli barang secara online menggunakan Dogecoin.", 
-    "dgc-stores-modal-sale": "Dogecoin aman selama pembayaran, karena kamu tidak memberikan informasi bank yang sensitif! Hanya mengirim Dogecoinmu ke alamat yang sudah ditentukan oleh merchant tersebut, dan barang kamu akan diproses dan siap untuk diantar dengan instant!", 
-    "dgc-stores-modal-businesses": "Berikut ini beberapa bisnis yang menerima Dogecoin:", 
+    "dgc-stores-modal-shop-online": "Kamu bisa berbelanja daring dengan Dogecoin.", 
+    "dgc-stores-modal-sale": "Dogecoin tergolong aman selama checkout, karena kamu tidak memerlukan informasi bank yang sensitif! Cukup kirim Dogecoinmu ke alamat yang ditentukan oleh pedagang, kemudian barangmu siap untuk diproses dan dikirim dalam sekejap!", 
```

Comment:
```
Cukup kirim Dogecoinmu ke alamat yang ditentukan oleh penjual
```

I think it's more common for people to use "penjual" instead of "pedagang" to describe online seller